### DISCUSSION TITLE
Need access to Oracle VM setup Google Doc link

### DISCUSSION THREAD

**Discussion by manojLondhe** [Sunbird-cQube/community#775] [Q&A] [answered]
Hello All,

CQube - [Oracle server setup doc](https://cqube.sunbird.org/installation-and-upgradation/step-wise-installation-process/for-oracle-server) has one [google document link](https://docs.google.com/document/d/1ODP13lUjO4OYV-rkFhPs-XMG0aehsfiWcCekkAN3Bfs/edit#heading=h.oqy188kfbwi5)

Anyone knows who owns it - can it be made public pls?

**Comment by surendrasinghs** [comment, marked as answer]
Please try now.

**Reply by manojLondhe** [reply]
can access now, thanks!

**Reply by manojLondhe** [reply]
There seems to be another one with link
https://docs.google.com/document/u/1/d/1-NH5bdHj-fGqXn-ztm7PhlM8MoZAEUNAYiG0z1bq7cY/
Need access to that as well
